### DISCUSSION TITLE
Using or converting .dae models to glft/glb and calling  multiple models from 1 gltf/glb file

### DISCUSSION THREAD

**Discussion by Biglow230** [google/model-viewer#2992] [Q&A]
This is a bit of a 2 part ask.

I'm working on a fan project and I have a ton of .dae models with animations.

The dae files are various animations for the same character. Run, jump, attack, etc.

The dirty way I've been putting all the animations together has been to open 1 main dae model in blender and copy the others animations one by one. Name them and export the finished file to a gltf.

My first question is, can you use .dae files without converting them to gltf/glb I know performance would take a hit but these are really low poly models.

I've tried mass exporting through threejs.org editor and the gltf/glb export feature wasn't responding. Is there a nother method for batch exporting to glft/glb?

My second question is, if I import multiple dae models with their respective animations and export as a gltf/glb is there a way to reference a specific model within the gltf/glb. Threejs has this functionality but I don't know if model-viewer can do this? If so I could just upload a master gltf/glb and call models directly from it rather than trying to export literally 500+ models.

Thank y'all for your time.

**Comment by WadeMorrison** [comment]
I haven't used this myself other than the blender exporter, but there is a list of other possible exporters that can be found [here](https://kcoley.github.io/glTF/)! I think the one you may be after might be [fbx2gltf](https://github.com/pissang/clay-viewer)? As for being able to use .dae files, I can't seem to be able to use them. I'm still pretty new to all of this, so I could be wrong, but I hope it at least helps!

**Reply by Biglow230** [reply]
Thank you for this resource, I see a batch converter to gltf 1.0, I'll have to try and see if this works. 

This resource should be pinned hahaha 😂

**Reply by WadeMorrison** [reply]
Haha glad I could have helped!
